src/components/common-components/ProjectModal/ProjectModal.js

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react';
+import Modal from "react-bootstrap/Modal";
+import { Button} from 'react-bootstrap';
+import Fade from 'react-reveal/Fade';
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./ProjectModal.css"
+
+/**
+ * @returns {React.Component}
+ */
+const ProjectModal = props => { 
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
```

Comment:
This is unused. Typescript/a linter will help you find unused variables like this